### PR TITLE
Add a note about "good first bug" procedure

### PR DESCRIPTION
**Pull request by potch**
@atsay suggested these potential tweaks for filing issues.

### CODE REVIEW

**Comment by EnTeQuAk** [comment]
Looks good to me, r+ - feel free to merge once tests are green :)

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -16,6 +16,11 @@ Before contributing code, please note:
 -  Follow `PEP8`_, `jshint`_ and our other `style guide conventions`_.
 -  Please write tests and read the docs on `addons-server`_.
 
+When filing an issue:
+
+-  Be as specific as you can with what you expected vs. what you experienced.
+-  Is this potentially a "good first bug"? Avoid insider jargon in your description, and mark the issue "maybe good first bug".
+
```
Comment (wagnerand) [inline CONTRIBUTING.rst]: Can we make it clear to use "maybe good first bug" **always**? Triaging folks don't look at "good first bug", and we want to make sure every good first bug has at least a mentor.

**Comment by atsay** [comment]
I'll make edits after Potch's go through.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (muffinresearch) [inline CONTRIBUTING.rst]: Afaik people outside the team can't add labels.

**Comment by diox** [comment]
Is this good to go ?

**Comment by wagnerand** [comment]
I'm not really sure who the target audience for this file is. If we want to show it when filing new issues, we need to convert it to Markdown. Do we want notes for staff in this file?

**Pull request merged**